docs/installation/local.md

```diff
@@ -21,6 +21,9 @@ Running this tool requires installing the nodejs as well as python dependencies.
 ```
 npm install -g bower
 bower install
+pip install virtualenv
+virtualenv venv
+source activate venv (venv\Scripts\activate for Windows)
```

Comment:
Perhaps put the Windows note in a comment so that copying and pasting the commands still works on Linux.
* source activate venv  # (venv\Scripts\activate for Windows)